specification/implementation/metadata/mets/mets-root/index.md

```diff
@@ -1,7 +1,9 @@
 ### 5.3.1.	Use of the METS root element (element mets)
 The purpose of the METS root element is to describe the container for the information being stored and/or transmitted. The implementation of the root element for a METS document conformant with CSIP uses attributes from the METS specification and attributes added for the purposes of the CSIP.
 
-In addition to the attributes the METS root element mets MUST define all relevant namespaces and locations of XML schemas using the `@xmlns` and `@xsi:schemaLocation` attributes.
+In addition to the attributes the METS root element mets MUST define all relevant used namespaces and locations of XML schemas using the `@xmlns` and `@xsi:schemaLocation` attributes.
```

Comment:
In addition to its attributes the METS root element mets MUST define all relevant namespaces and XML schema locations used in the package employing the `@xmlns` and `@xsi:schemaLocation` attributes.

Reply:
Fixed